node/libs/roles/src/validator/conv.rs

```diff
@@ -57,7 +57,7 @@ impl ProtoFmt for Genesis {
                         .context("validators")?,
                     0,
                 )
-            // empty validator set
+            // empty validator set, will raise an exception in Committee:new()
```

Comment:
there is no "raising exceptions" in rust. Use "Committe::new() will return an error" or similar.